modules/recording/recording/report.tsx

```diff
@@ -48,7 +48,21 @@ const Report = () => {
     setActiveTab(tab);
   };
 
-  // TODO: Refactor as a resource
+  const componentRef = useRef<HTMLDivElement>(null);
+  const [hideIcons, setHideIcons] = useState(false)
+  const downloadPdf = useReactToPrint({
+    content: () => componentRef.current,
+    documentTitle: 'Report',
+    onAfterPrint: () => setHideIcons(false),
```

Comment:
The magic to hide the icons :D

I would expect some CSS but i think that works lol

Reply:
I tried couple of solution and the weren't working and it was taking time I decided to use that approach